Dockerfile

```diff
@@ -14,19 +14,18 @@ RUN apk update && \
 WORKDIR /vitalam-identity-service
 
 # Install base dependencies
+RUN npm -g install npm@latest
+
 COPY . .
-RUN npm install
+RUN npm ci install
```

Comment:
Its just `npm ci`

Reply:
literally

```
RUN npm ci --production
```

is what we want here I believe